components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -237,6 +260,22 @@ export default function () {
                     </Alert>
                 )}
             </div>
+            <div>
+                <h3 className="mt-12">Delete Project</h3>
```

Comment:
**suggestion**: What do you think of using a different verb here to avoid causing any confusion whether this is also going to delete a project or repository on the provider?

```suggestion
                <h3 className="mt-12">Remove Project</h3>
```